### PR TITLE
Feature/show2 1704 add navbar everywhere

### PR DESCRIPTION
**Pull request by alantoa**
# Why 

Add navbar for everywhere even if without login. 

**Note: This PR does not include showing the Trending screen when not logged in. It will be included in the next PR.** cc @hirbod  

# How
- Update logic for displaying bottom tabs.
- Update user icon. If the user is not logged in, replace the avatar image with the user icon.

# Test Plan

Native: 

https://user-images.githubusercontent.com/37520667/233324095-fb73c4e1-8dbc-41ec-bc36-8310ba928ec8.mp4

Web: 

https://user-images.githubusercontent.com/37520667/233324124-f7208377-1fc5-41a2-9618-cef7f9ba4b5f.mp4

 *

### CODE REVIEW

File: packages/app/components/feed.tsx [hunk 1]
```diff
@@ -25,181 +23,14 @@ export const Feed = () => {
...
       {Platform.OS !== "web" && (
         <SwipeListHeader canGoBack={false} withBackground />
       )}
-      <SwipeList bottomPadding={bottomPadding} data={data} />
+      <SwipeList bottomPadding={bottomBarHeight} data={data} />
     </View>
   );
 };
-
-// export const FeedList = () => {
```
Comment (alantoa) [inline packages/app/components/feed.tsx:43]: Remove this abandoned code because we no longer use it.

**Comment by hirbod** [comment]
Bildschirmfoto 2023-04-20 um 14 45 16

Sign In Button on web looks wrong

File: packages/app/navigation/tab-bar-icons.tsx [hunk 2]
```diff
@@ -161,10 +163,16 @@ export const TrendingTabBarIcon = ({ color, focused }: TabBarIconProps) => {
 export const NotificationsTabBarIcon = ({
   color,
   focused,
-  onPress,
 }: TabBarIconProps) => {
+  const redirectToScrssn = useRedirectToScreen();
```
Comment (hirbod) [inline packages/app/navigation/tab-bar-icons.tsx]: Typo: Scrssn

File: packages/app/navigation/tab-bar-icons.tsx [hunk 3]
```diff
@@ -161,10 +163,16 @@ export const TrendingTabBarIcon = ({ color, focused }: TabBarIconProps) => {
...
   return (
-    <TabBarIcon tab="/notifications" onPress={onPress}>
+    <TabBarIcon
+      onPress={() =>
+        redirectToScrssn({
+          pathname: "/notifications",
```
Comment (hirbod) [inline packages/app/navigation/tab-bar-icons.tsx:172]: Typo

File: packages/app/navigation/tab-bar-icons.tsx [hunk 4]
```diff
@@ -195,13 +203,28 @@ const UnreadNotificationIndicator = () => {
   );
 };
 
-export const ProfileTabBarIcon = () => {
-  const { user } = useUser();
+export const ProfileTabBarIcon = ({ color }: TabBarIconProps) => {
+  const { user, isAuthenticated } = useUser();
   const { userAddress } = useCurrentUserAddress();
+  const redirectToScrssn = useRedirectToScreen();
```
Comment (hirbod) [inline packages/app/navigation/tab-bar-icons.tsx]: Typo

File: packages/app/navigation/tab-bar-icons.tsx [hunk 5]
```diff
@@ -195,13 +203,28 @@ const UnreadNotificationIndicator = () => {
...
 
   return (
-    <TabBarIcon tab={`/@${user?.data?.profile?.username ?? userAddress}`}>
-      <Avatar url={user?.data?.profile?.img_url} alt={"Profile Avatar"} />
+    <TabBarIcon
+      onPress={() =>
+        redirectToScrssn({
```
Comment (hirbod) [inline packages/app/navigation/tab-bar-icons.tsx]: Typo

**Review by hirbod — CHANGES_REQUESTED** [review]

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

**Review by hirbod — APPROVED** [review]

**Pull request merged**